### PR TITLE
Move all static text into the language file

### PR DESCRIPTION
**Pull request by foxted**
Moved all static text into files: 

- [x] cachet.php (for application front-end text)
- [x] dashboard.php (for all dashboard texts)
- [x] forms.php (for all the forms fields)
- [x] errors.php (for the errors pages, 404 only at this time)
- [x] setup.php (for setup-related text)

I couldn't do this for the other languages because I don't speak them. I could have done it for the french but the translation is still at the validation stage on CrowdIn, but let me know if you'd be interested.

PS: Hope I understood correctly the issue [#230](https://github.com/cachethq/Cachet/issues/230), otherwise let me know what I can do to help :)

### CODE REVIEW

**Comment by jbrooksuk** [comment]
Really good work @foxted - can you fix up the StyleCI issues and then I'll take a look through everything.

**Comment by foxted** [comment]
Do you know where I can see the details regarding the style issues?

**Comment by jbrooksuk** [comment]
Download [this patch](https://styleci.grahamjcampbell.co.uk/commits/27a35bcdb18d87853033f9c2c993f236b0da74e1/diff) and apply it to your branch, commit and push :)

File: app/views/dashboard/incidents/edit.blade.php [hunk 1]
```diff
@@ -13,43 +13,30 @@
...
-                        {{ trans('cachet.incident.status')[4] }}
-                    </label>
+                    <label for="incident-name">{{ trans('forms.incidents.status') }}</label><br />
+                    @foreach(trans('cachet.incidents.status') as $statusID => $status)
+                        <label class="radio-inline">
+                            <input type="radio" name="incident[status]" value="{{ $statusID }}">
+                            <i class="icon ion-flag"></i>
+                            {{ $status }}
+                        </label>
+                    @endforeach
```
Comment (Crash--) [inline app/views/dashboard/incidents/edit.blade.php]: With this, we'll only have one icon image.

Comment (GrahamCampbell) [inline app/views/dashboard/incidents/edit.blade.php]: Yeh, what's going on here with the trans call in the foreach loop?

Comment (foxted) [inline app/views/dashboard/incidents/edit.blade.php]: I didn't notice! I'll fix this.

**Comment by GrahamCampbell** [comment]
Some of this alignment is very odd too.

**Comment by GrahamCampbell** [comment]
And, actually, all the non-english files need syncing up.

**Comment by jbrooksuk** [comment]
@GrahamCampbell do you mean on CrowdIn?

**Comment by GrahamCampbell** [comment]
No, I mean in this actual repo.

**Comment by jbrooksuk** [comment]
How do you mean syncing up? Is it not right?

**Comment by GrahamCampbell** [comment]
I was meaning the layout of the english files have changed, so the identical changes need to be made to the other languages.

**Comment by foxted** [comment]
They were already out of sync. Like I said, I don't speak the other languages, but I can sync the files and leave them blank waiting for translation (not sure if it will be useful though)

**Comment by GrahamCampbell** [comment]
Meh, just leave it. I'll sync them up later.

**Comment by foxted** [comment]
Ok.

**Comment by GrahamCampbell** [comment]
Needs rebasing.

**Comment by jbrooksuk** [comment]
@foxted hold up rebasing until #196 is merged.

**Comment by foxted** [comment]
No problem, I'll wait ;)

**Comment by GrahamCampbell** [comment]
NB, it's been merged now. :)

**Comment by foxted** [comment]
Any advice about using StyleCI? Because I can't find anything regarding the issues, so I can't fix them :/ (I am a newbie at contributing on GitHub)

**Comment by GrahamCampbell** [comment]
Just append `/diff` to the url. It'll show you the diff.

**Comment by GrahamCampbell** [comment]
https://styleci.grahamjcampbell.co.uk/commits/3c961a8203cbd84d78b3f5296a91480ba2e72340/diff

**Comment by GrahamCampbell** [comment]
> Any advice about using StyleCI?

We follow PSR-2, and we also enforce a few rules from symfony's coding standards, and we force short array syntax and aligned arrays. The source code for all of this is available at https://github.com/StyleCI.

**Comment by foxted** [comment]
Thank you :)

File: app/lang/en/dashboard.php [hunk 2]
```diff
@@ -0,0 +1,111 @@
+<?php
+
+return [
+
+    'dashboard'         => 'Dashboard',
```
Comment (GrahamCampbell) [inline app/lang/en/dashboard.php]: why is this indented soo much?

Comment (foxted) [inline app/lang/en/dashboard.php]: I have a 4-spaces settings and auto-alignment on array items in my PHPStorm :/

**Comment by GrahamCampbell** [comment]
Hmm. There's something gone wrong with your rebasing here. Perhapse you didn't force push. I'll try an get this sorted...

**Comment by GrahamCampbell** [comment]
Where your changes to the css and migrations intentional?

**Comment by foxted** [comment]
I didn't touch any CSS or migration file, so I guess something went wrong with the rebasing (first time :/)

**Comment by GrahamCampbell** [comment]
Ok, moved to here: https://github.com/cachethq/Cachet/pull/277.

**Comment by GrahamCampbell** [comment]
Thanks so much for doing this. :) Is there any chance you could give that PR a quick check that everything is working, and that we haven't lost any recently added stuff.

**Pull request closed without merging**